public/static/locales/en/tools.json

```diff
@@ -0,0 +1,123 @@
+    "volumesWarning": "WARNING: Do not add Container Volumes or Container Volumes From settings to tools unless it is certain that tool is authorized to access that data.",
+    "makePublic": "Make Public",
+    "public": "Public",
+    "noTools": "No tools",
+    "allTools": "All",
+    "myTools": "Only my tools",
+    "publicTools": "Public tools",
+    "requestToolMI": "Request Tool",
+    "edit": "Edit",
+    "delete": "Delete",
+    "useToolInApp": "Use in app",
+    "shareWithCollaborators": "Share",
+    "tools": "Tools",
+    "share": "Share",
+    "searchTools": "Search tools",
+    "refresh": "Refresh",
+    "imageNameHelp": "If the image is in Docker Hub, this field should be in username/image-name format, where username is your Docker Hub username. If it's in another registry, such as the CyVerse registry, it should be in registry-host/image-name format.",
+    "validationErrMinCPUsGreaterThanMax": "Must be less than Max CPU Cores",
+    "validationErrMaxCPUsLessThanMin": "Must be greater than Min CPU Cores",
+    "validationErrMinRAMGreaterThanMax": "Must be less than Max Memory",
+    "validationErrMaxRAMLessThanMin": "Must be greater than Min Memory",
+    "validationErrMustBePositive": "Must be at least 0",
+    "newToolRequestDialogHeading": "New Tool Request",
+    "submit": "Submit",
+    "toolNameLabel": "What is the name of the tool?",
+    "toolDescLabel": "Briefly describe the tool",
+    "toolSrcLinkLabel": "Provide a link for your tool's source (GitHub, BitBucket, DockerHub etc...)",
+    "toolVersionLabel": "What is your tool's version?",
+    "toolDocumentationLabel": "Provide a link for your tool's documentation",
+    "toolInstructionsLabel": "Enter any other instructions or relevant information",
+    "toolTestDataLabel": "Provide a link for your tool's test data",
+    "validationErrMsgURL": "A valid URL must begin with either ftp or http or https.",
+    "toolAttributionLabel": "Attribution",
+    "descriptionLabel": "Description",
+    "restrictionsLabel": "Restrictions",
+    "memoryLimitLabel": "Memory Limit (bytes)",
+    "pidsLimitLabel": "PIDs Limit",
+    "secondsLimitLabel": "Maximum Run Time (seconds)",
+    "networkingLabel": "Networking",
+    "toolInformationLbl": "Tool Information",
+    "appsUsingToolLbl": "Apps using this tool",
+    "imageNameLbl": "Image Name",
+    "imageTagLbl": "Image Tag",
+    "imageUrlLbl": "Image URL",
+    "deprecatedLbl": "Deprecated",
+    "entryPointLbl": "Entry Point",
+    "uidLbl": "UID",
+    "workingDirLbl": "Working Directory",
+    "versionLbl": "Version",
+    "detailsLbl": "Details",
+    "resourceRequirementsLbl": "Resource Requirements",
+    "minCPUCoresLbl": "Minimum CPU Cores",
+    "maxCPUCoresLbl": "Maximum CPU Cores",
+    "minMemoryLimitLbl": "Minimum Memory",
+    "minDiskSpaceLbl": "Minimum Disk Space (bytes)",
+    "enabled": "Enabled",
+    "ariaCheckbox": "{{label}} checkbox",
+    "ariaTableListing": "Tool table listing",
+    "status": "Status"
```

Comment:
It'd be nice to sort these lines.  Not sure about VS Code, but I know Atom has an extension that will auto-sort for you.